src/vaishnavi/A15_ArrayExamples.java

```diff
@@ -0,0 +1,89 @@
+package vaishnavi;
+
+public class A15_ArrayExamples {
+
+	int printMaximumNumFromArray(int[] num) {
+		int maximumNumber =num[0];
+		for(int index = 0; index < num.length; index++) {
```

Comment:
If you are taking the 0th index as max number then you can start the loop from index 1 as well, same for other methods